src/Components/CryptoChart/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import axios from "axios";
+import { Line, Bar } from "react-chartjs-2";
+import { Chart, Filler } from "chart.js";
+import {Chart as ChartJS, CategoryScale, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend} from "chart.js";
```

Comment:
import {Chart as ChartJS, CategoryScale, `Filler`, LinearScale, PointElement, LineElement, BarElement, Title, Tooltip, Legend} from "chart.js";

you are importing chart twice from chart.js